composer.json

```diff
@@ -108,7 +108,7 @@
         "analyse": "psalm",
         "check-style": "ecs check",
         "fix-style": "ecs check --fix",
-        "phpunit": "phpunit",
+        "phpunit": "phpunit --exclude-group=functional",
```

Comment:
I think it's easier to use suites, don't you? I think we did the same in the order edit plugin